frontend/src/component/admin/auth/SamlAuth/SamlAuth.tsx

```diff
@@ -98,6 +100,24 @@ export const SamlAuth = () => {
         <>
             <Grid container sx={{ mb: 3 }}>
                 <Grid item md={12}>
+                    <ConditionallyRender
+                        condition={Boolean(samlConfiguredThroughEnv)}
+                        show={
+                            <Alert sx={{ mb: 2 }} severity='warning'>
+                                SAML is currently controlled via environment
+                                variables. Please see the{' '}
+                                <a
+                                    href='https://www.unleash-hosted.com/docs/enterprise-authentication'
+                                    target='_blank'
+                                    rel='noreferrer'
+                                >
+                                    documentation
+                                </a>{' '}
+                                to learn how to use environment variables for
+                                configuring SAML
```

Comment:
Nitpicking but I noticed this sentence is not consistent with the one in OIDC. We may want to make them consistent.

How about something like: `{OIDC | SAML} is currently configured through environment variables. Please refer to the documentation for detailed instructions on how to set up {OIDC | SAML} using these variables.`

Reply:
Yeah. Will fix